pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "pycep-parser"
-version = "0.4.1"
+version = "0.4.1a"
```

Comment:
It has to be changed probably

Reply:
no need, please remove the change here. A preview release will be automatically created and published to `pypi` when merged to the master branch 😄

Reply:
Agree, next commit will bring this change